daemon/Lokinet.modulemap.in

```diff
@@ -0,0 +1,4 @@
+
+module LokinetExtension {
+       header "@CMAKE_SOURCE_DIR@/include/lokinet-extension.hpp"
+}
```

Comment:
This header doesn't exist anymore so I'm pretty sure we can remove this modulemap.